lib/libc/string/memchr.c

```diff
@@ -33,14 +33,24 @@ __FBSDID("$FreeBSD$");
 #define ONES ((size_t)-1 / UCHAR_MAX)
 #define HIGHS (ONES * (UCHAR_MAX / 2 + 1))
 #define HASZERO(x) (((x)-ONES) & ~(x)&HIGHS)
+#define MIN(a, b) ((a) < (b) ? (a) : (b))
```

Comment:
sys/param.h has a definition of this, surprised it doesn't give a duplicate warning